package.json

```
@@ -160,7 +160,7 @@
     "webpack-dev-server": "^4.11.1"
   },
   "dependencies": {
-    "@apollo/client": "^3.8.8",
+    "@apollo/client": "^3.7.14",
```

If this is the version that works for us, then I suggest we pin it in place until further notice.

```suggestion
    "@apollo/client": "3.7.14",
```

---

good idea!